gctscript/vm/manager.go

```diff
@@ -0,0 +1,123 @@
+package vm
+
+import (
+	"errors"
+	"fmt"
+	"path/filepath"
+	"sync"
+	"sync/atomic"
+
+	"github.com/thrasher-corp/gocryptotrader/common"
+	"github.com/thrasher-corp/gocryptotrader/engine/subsystem"
+	"github.com/thrasher-corp/gocryptotrader/log"
+)
+
+const gctscriptManagerName = "GCTScript"
+
+// GctScriptManager loads and runs GCT Tengo scripts
+type GctScriptManager struct {
+	config   *Config
+	started  int32
+	stopped  int32
+	shutdown chan struct{}
+	// Optional values to override stored config ('nil' if not overridden)
+	MaxVirtualMachines *uint8
+}
+
+// NewManager creates a new instance of script manager
+func NewManager(config *Config) (*GctScriptManager, error) {
```

Comment:
I think the last thing I'd like is that this file gets test coverage

Reply:
Done